### DISCUSSION TITLE
recent problem with version 4.4.2 on SeaMonkey 2.53.18.1

### DISCUSSION THREAD

**Discussion by dhdurgee** [kewisch/gdata-provider#734] [Support]
I had been using the final version of this add-on for SeaMonkey for the past three years without problems.  That changed a week ago when I had an unexpected shutdown of my Linux mint 20.3 laptop with SeaMonkey open.

When I restarted the laptop and opened SeaMonkey I got the expected restore session prompt, but then it turned out that my logins.json file had been corrupted.  I shut SeaMonkey down and replaced the corrupted file with an older one.  That corrected most of my problems, except that I had to recreate the OAuth configurations for my email accounts.

The problem I have now is that the Provider will NOT accept and retain my sign-on to Google for some reason.  I guess there was some other damage than logins.json that accounts for it.  At this point I had to disable the add-on, as an event caused a notification that cannot be dismissed as the login prompt keeps coming up every time I attempt to dismiss it.

I see that there are user_pref entries in prefs.js that relate to the calendar as well as the calendar-data directory and its contents and I understand there are cookies associated with this add-on as well.

I have tried to use debugging without success by enabling the two debug settings, but nothing gets logged.

Can anyone help me get this back into proper operation?  I will be happy to provide more information if needed.  I will remove lines from prefs.js or cookies if that is what is needed to get this working.

I have posted some additional information to the alt.comp.software.seamonkey newsgroup about my attempts to debug and what I am encountering.

Thank you for any possible assistance.

Dave

**Comment by kewisch** [comment]
Unfortunately SeaMonkey is no longer supported with this add-on. If you don't want to switch to Thunderbird then I'd suggest SeaMonkey's support channels.
